gammapy/scripts/image_pipe.py

```diff
@@ -175,13 +222,102 @@ def make_maps(self, radius, bkg_norm=True):
             Disk radius in pixels for the significance map.
         bkg_norm : bool
             If true, apply the scaling factor to the bkg map
+        spectral_index : float
+            Assumed power-law spectral index
+        exposure_weighted_spectrum : bool
+            True if you want that the total excess / exposure gives the integrated flux
         """
         self.make_total_counts()
         self.make_total_bkg(bkg_norm)
         self.make_significance(radius)
+        self.make_total_excess()
+        self.make_total_exposure(spectral_index, exposure_weighted_spectrum)
 
     def make_psf(self):
         log.info('Making PSF ...')
 
-    def make_exposure(self):
-        log.info('Making EXPOSURE ...')
+    def make_total_excess(self):
+        """Compute excess between counts and bkg map."""
+        self.maps["total_excess"] = SkyMap.empty_like(self.empty_image)
+        excess = self.maps["total_counts"].data - self.maps["total_bkg"].data
+        self.maps["total_excess"].data = excess
+
+    def make_exposure(self, obs_id, spectral_index=2.3, exposure_weighted_spectrum=False):
+        """Compute the exposure map for one observation.
+
+        Excess/exposure will give the differential flux at the energy Eref at the middle of the self.energy_band
```

Comment:
I think it's simpler and easier to understand to always use `energy_ref = 1 TeV` and `flux_ref = 1 cm^-2 s^-1 TeV^-1` here. So the user gets back exposure, uses `flux = excess / exposure` to compute the differential flux at 1 TeV, and then they can compute a differential flux at any energy they like, or an integral flux in any energy band they like.
There are cases where users don't want the integral flux in the measurement energy band, but e.g. above 1 TeV (that's what we do for the flux maps we release for HGPS). 

But really it's a matter of taste. If the caller wants the differential flux and the energy band mid-point, then yes, returning exactly that is convenient.

So ... I'll let you make the call here (for today ... will re-discuss with Axel and others).

One more point: the `exposure_weighted_spectrum` option name isn't very good, no?
Also if it's false, the exposure computed is a power-law spectrum-weighted value, right?

So either rename to `for_integral_flux` or something with a better name.
Or -- if you follow my suggestion from above to always return diff flux at 1 TeV and let the caller choose a different reference energy or integral energy band, then this option can be removed.